charts/oes/values.yaml

```diff
@@ -158,6 +157,50 @@ global:
   ## Use the below flag to exclude pre-delete helm hooks to the
   ## rendered manifests
   preDeleteHelmHooks: true
+  # Minio access/secret keys for the in-cluster S3 usage
+  # Minio is not exposed publically
+  minio:
```

Comment:
minio configuration can stay outside global as none of the other subcharts are using minio configuration